app/views/suggestions/_suggestion.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> <%=@heading %> </h1>
+
+<% Food.hashify(@data).each do |suggestion| %>
```

Comment:
Okay, I see why you did this, so you can reuse this partial. That totally makes sense. Maybe you could put a comment above this section to explain what's going on.